src/firestore/encodeFirestoreValue.ts

```diff
@@ -57,5 +57,8 @@ function encodeHelper(val: any): any {
 }
 
 export function encodeFirestoreValue(data: any): { [key: string]: any } {
-  return _.mapValues(data, encodeHelper);
+  return Object.entries(data).reduce((acc, [key, val]) => {
+    acc[key] = encodeHelper(val);
+    return acc;
+  }, {} as { [key: string]: any });
```

Comment:
I've liked doing `Record<string, any>` more than this nonsense of an object - but I'll leave it up to you if you want to adjust it